app/src/main/res/values/strings.xml

```diff
@@ -20,4 +23,7 @@
     <!-- One Value Widget -->
     <string name="one_value_image_description">Cloud symbol</string>
     <string name="one_value_error_text">Error</string>
+
+    <string name="mapbox_token">## enter your mapbox token here ##</string>
```

Comment:
What is the most elegant way to store security tokens of Android projects on GitHub?

Reply:
In my opinion, the most elegant way is to store it token in the local gradle properties and include the stored value in the build process. The required changes are documented in #5. I start with this issue as soon as this pull request is merged.